app/enterprise/1.3-x/deployment/installation/amazon-linux-2.md

```diff
@@ -0,0 +1,342 @@
+    | TYPE | DATABASE | USER | ADDRESS      | METHOD |
+    | # IPv4 local connections:                      |
+    | host | all      | all  | 127.0.0.1/32 |  md5   |
+    | # IPv6 local connections:                      |
+    | host | all      | all  | ::1/128      | ident  |
+  ```
+
+3. Save and exit the file, then restart Postgres
+  ```
+  $ sudo systemctl restart postgresql
+  ```
+
+Now that the database is set up, Kong must be given credentials to access the database. This is done by adding the Postgres user and password information to Kong’s configuration file.
+
+
+1. Make a copy of Kong’s configuration file
+  ```
+  sudo cp /etc/kong/kong.conf.default kong.conf
+  ```
+
+2. Open the new configuration file
+  ```
+  sudo vi /etc/kong/kong.conf
+  ```
+
+3. Search for `pg_password` under `DATASTORE,` uncomment `pg_password` and add the value. 
+  ```
+  #pg_user = kong
+  pg_password = kong
+  #pg_datastore = kong
+  ```
+  > Note: If you used different values for the user and database name, uncomment the pg_user and pg_datastore variables and add the username and datastore values
+
+
+## Step 4. Customize Your Configuration
+Before starting Kong, you can further modify Kong’s configuration file to enable a host of different features:
+
+
+### Enable RBAC
+
+  Kong Enterprise allows applying RBAC to all requests. To enable RBAC, you must set RBAC to `on`, select an authentication plugin for Kong Manager, and configure the [Session plugin](/hub/kong-inc/session/).
+
+  ```
+  enforce_rbac = on
+
+  admin_gui_auth = basic-auth
+
+  admin_gui_session_conf = {"secret":"password", "cookie_secure": false, "cookie_samesite": "off", "cookie_name": "admin_session", "storage": "kong"}
+  ```
+
+  Export the KONG_PASSWORD environment variable:
+  ```
+  export KONG_PASSWORD=kong
+  ```
+
+### Enable Kong Manager
+
+  Kong Enterprise's Graphical User Interface, **Kong Manager**, can be connected by setting the `admin_gui_url` property to the EC2 instance's `IPv4` address
+
+  ```
+  admin_gui_url = http://ec2-xx-xxx-xx-xx.us-east-2.compute.amazonaws.com:8002
+  ```
+
+### Enable the Dev Portal
+
+  Kong Enterprise's **Developer Portal** can be connected by setting the `portal` property to `on` and setting the `portal_gui_host` property to the EC2 instance's `IPv4` address
+
+  ```
+  portal = on
+  portal_gui_host = http://ec2-xx-xxx-xx-xx.us-east-2.compute.amazonaws.com:8003
+  ```
+
+
+## Step 5. Start Kong
+
+1. Run migrations
+  ```
+  kong migrations bootstrap -c /etc/kong/kong.conf
+  ```
+2. Start Kong
+  ```
+  sudo /usr/local/bin/kong start -c /etc/kong/kong.conf
+  ```
+
+
+## Step 6. Verify Kong is Receiving Requests
+
+##### Install HTTPie to easily send requests to Kong *(optional)*
+  ```
+  sudo yum install python3
+
+  sudo pip3 install --upgrade pip setuptools
+
+  sudo /usr/local/bin/pip3 install --upgrade httpie
+  ```
+
+#### Test your connection to Kong
+
+**httpie**
+  ```
+  http :8001/services 
+  
+  if RBAC is enabled:
```

Comment:
not a bad idea to explain how to do this with RBAC on, but we should also link to another section in deployment guide where we enable RBAC. I dont think this belongs in the standard install doc though because we want people to install and start using quickly without having to worry about rbac.